index.js

```diff
@@ -28,11 +30,38 @@ async function activate() {
 
   loadConfig(path);
 
-  if (!validateConfig() || !isTrackingAllowed()) {
+  if (!validateConfig()) {
     return;
   }
 
-  await trackUsage('vscode-extension', 'ACTIVATION');
+  if (isTrackingAllowed()) {
+    try {
+      await trackUsage(
+        'vscode-extension-interaction',
+        'ACTIVATION',
+        {},
+        getAccountId()
+      );
+    } catch (e) {
+      console.log(e);
+    }
+  }
+
+  if (vscode.workspace.getConfiguration('hubl').get('lint.enabled')) {
```

Comment:
Just a suggestion (feel free to ignore), but you could move `hubl` and also `lint.enabled` to constants since they're referenced in a few places.